templates/addons/env/rds-postgres.yml

```diff
@@ -40,7 +40,7 @@ Resources:
     Type: AWS::RDS::DBSubnetGroup
     Properties:
       DBSubnetGroupDescription: Group of subnets to place DB into
-      SubnetIds: [ !Ref PrivateSubnet1, !Ref PrivateSubnet2 ]
+      SubnetIds: !Split [ ",", !Ref PrivateSubnets ]
```

Comment:
We missed this one the other day @lgarvey 😬

Reply:
Ouch!

Reply:
It kind of demonstrates the use case for automated checks 😜

Reply:
Why wasn't it picked up by the checks?

Reply:
They were not running the pre-commit or the pipeline, this was picked up when I added them in.